models/applicationSubmissionsModel.js

```diff
@@ -28,35 +35,23 @@ async function insert(submission) {
 
 async function update(id, decision) {
   return db('application_submissions')
+    .update({ decision })
     .where({ id })
-    .update({decision})
     .returning('*');
 }
 
-async function acceptAndDenyAllOthers(id, skilled_impact_request_id) {
-  await db.transaction(transaction => {
-    db('application_submissions')
-      .where({ id })
-      .update({ decision : 'ACCEPTED' })
-      .transacting(transaction)
-      .then(async () => {
-        await db('application_submissions')
-          .where({ skilled_impact_request_id })
-          .where('id', '<>', id)
-          .update({ decision: 'DENIED' })
-          .transacting(transaction);
-      })
-      .then(transaction.commit)
-      .catch(transaction.rollback);
-  })
-  
-  return findById(id);
+async function updateAll(ids, decision) {
```

Comment:
RIP `acceptAndDenyAllOthers`

Reply:
The function that we needed but didn't deserve